app/views/categories/show.html.erb

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<head>
```

Comment:
Can we add the Category title to the head of the page? I know the URL will have that information, yet it could be helpful to the user to see the category as a header.  @samuelssnider